templates/cloud/hybrid-cloud.html

```diff
@@ -0,0 +1,409 @@
+			<p>Hybrid clouds enable organisations to optimise their infrastructure costs by always running their workloads where it makes more sense from an economical standpoint.</p>
+			<p>
+        <a href="/engage/hybrid-cloud-business-guide">Read the whitepaper - “A business guide to hybrid cloud”&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+    <div class="col-5 u-hide--small u-vertically-center u-align--center">
+			{{ image (
+				url="https://assets.ubuntu.com/v1/45120edb-Hybrid+Cloud.svg",
+				alt="Hybrid Cloud",
+				width="324",
+				height="255",
+				hi_def=True,
+				loading="auto"
+				) | safe
+			}}
+    </div>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="row u-equal-height">
+    <div class="col-5">
+			<h2>How does a hybrid<br/> cloud work?</h2>
+			<h4>Since the hybrid cloud is a cloud computing architecture that uses existing public and private cloud infrastructure, the way it works is not significantly different from how public and private clouds work on their own.</h4>
+    </div>
+    <div class="col-7">
+			<p>The hybrid cloud leverages virtualisation, containerisation, software-defined networking (SDN) and software-defined storage (SDS) technologies to divide physical resources, aggregated into pools, and allocate them to virtual machines (VMs) or containers. Dedicated <a href="/openstack/what-is-openstack">management software</a> is used to aggregate compute, network and storage resources, distributed across the data centre, and enable on-demand provisioning of virtualised resources through a self-service portal.</p>
+			<p>What differentiates hybrid cloud from other cloud architectures is  the additional layer of software, running on top of existing clouds. Known as the HCM, this software enables <a href="https://juju.is" class="p-link--external">deploying, integrating and migrating workloads between private and public clouds</a> depending on capacity and cost conditions. Since <a href="/engage/cloud-pricing-report-2021">private clouds prove to be more cost-effective than public clouds</a> when running workloads in the long term and at scale, the majority of the workloads should always be run in the private cloud. However, organisations can move their workloads to the public cloud during heavy load periods. HCM enables this migration across various cloud providers, ensuring maximum CAPEX and OPEX efficiency.</p>
+		</div>
+	</div>
+</section>
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-5">
+			<h2>Hybrid cloud vs<br/> multi-cloud</h2>
+			<h4>Hybrid cloud and <a href="/cloud/multi-cloud">multi-cloud</a> are two mutually exclusive terms that are often confused.</h4>
```

Comment:
Again if you want to merge this hybrid cloud page first I would remove this link for now as it's a 404 until that page is built. In the PR where you build that page add it back